src/db/commit.fbs

```diff
@@ -22,7 +22,7 @@ table LocalMeta {
 // Commit metadata specific to snapshot commits.
 table SnapshotMeta {
     last_mutation_id: ulong;
-    cookie: string;
+    cookie_json: [ubyte];
```

Comment:
I'm curious why we prefer `[ubyte]` over string?

Reply:
For better or worse I was following the convention established by the args: https://github.com/rocicorp/repc/blob/962d45b714dd285628855887fa656baceb386e9c/src/db/commit.fbs#L18. I suspect since it's bytes under the hood, that's why someone chose [ubyte].